var/spack/repos/builtin/packages/py-pyglet/package.py

```diff
@@ -11,12 +11,28 @@ class PyPyglet(PythonPackage):
     for developing games and other visually rich applications.
     """
 
-    homepage = "https://github.com/pyglet/pygle://github.com/pyglet/pyglet"
-    pypi = "pyglet/pyglet-1.4.2.tar.gz"
+    homepage = "https://github.com/pyglet/pyglet"
+    pypi = "pyglet/pyglet-2.0.9-py3-none-any.whl"
```

Comment:
We don't use wheels in Spack but build from source. 
Since the changed zip starting from 1.4.3 you can distinguish between `zip` and `tar.gz` in `url_for_version` further down.
Keep in mind to fix the checksums and dependencies as well for the newer versions then.

Reply:
Good point.

I'll put the PR in Draft status for now though: https://github.com/pyglet/pyglet/issues/999

Reply:
Could also delete the pyproject.toml from those versions using os.remove so you don't have to use GitHub.